svn/common.py

```diff
@@ -54,8 +54,9 @@ def run_command(self, subcommand, args, success_code=0,
                              stderr=subprocess.STDOUT,
                              env={'LANG': 'en_US.UTF-8'})
 
-        stdout = p.stdout.read()
         r = p.wait()
+        stdout = p.stdout.read()
```

Comment:
The reason that this preceded the `wait()` call, before, was because, as a matter of convention, the process might write a ton of screen output and it won't all build-up in the buffer that way. Thoughts?